crates/iota-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1581,7 +1666,7 @@ impl PgIndexerStore {
                 .select(epochs::network_total_transactions)
                 .get_result::<Option<i64>>(conn)
         })
-        .context("Failed to get network total transactions in epoch")
+        .context(format!("Failed to get network total transactions in epoch {epoch}").as_str())
```

Comment:
lowercase for erro messages 

```suggestion
        .context(format!("failed to get network total transactions in epoch {epoch}").as_str())
````